frontend/src/App.js

```diff
@@ -8,6 +9,9 @@ function App() {
           <Route path={'/'}exact>
             <HomePage />
           </Route>
+            <Route path={'/Instructions'}exact>
```

Comment:
```suggestion
            <Route path={'/instructions'}exact>
```
i would use a small path name